internal/service/codebuild/fleet_data_source.go

```diff
@@ -211,7 +211,7 @@ func dataSourceFleetRead(ctx context.Context, d *schema.ResourceData, meta inter
 	d.Set("overflow_behavior", fleet.OverflowBehavior)
 
 	if fleet.ScalingConfiguration != nil {
-		if err := d.Set("scaling_configuration", []interface{}{flattenScalingConfiguration(fleet.ScalingConfiguration)}); err != nil {
+		if err := d.Set("scaling_configuration", flattenScalingConfiguration(fleet.ScalingConfiguration)); err != nil {
```

Comment:
Just to clarify as it's unrelated to the other `input` changes - was this some other issue uncovered during acceptance testing?

Reply:
Yeah, it fixes a panic

> scaling_configuration.0: '' expected a map, got 'slice'

`flattenScalingConfiguration` was updated in the resource, but the data source was never tested